src/main/java/school/lemon/changerequest/java/generics/pair/TestingApp.java

```diff
@@ -0,0 +1,32 @@
+package school.lemon.changerequest.java.generics.pair;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+
+/**
+ * Created by Yaroslav Pavlinskiy on 22.12.2016.
+ */
+public class TestingApp {
```

Comment:
Your Demo class should provide demo for all possible operations with 2 or 3 different pair types ( for example: <String,Integer>, <Double, Float>, <Integer, Container>)